protocol/x/prices/keeper/util_test.go

```diff
@@ -178,7 +190,7 @@ func TestIsTowardsIndexPrice(t *testing.T) {
 			indexPrice:     2,
 			expectedResult: true,
 		},
-		"Towards: idx < curr == new": {
+		"Towards: idx < new == curr": {
```

Comment:
test result easier to infer with test case renames: should be towards when new is between index and curr, inclusive.